.github/workflows/cursor-pr-review-agent.yml

```
@@ -0,0 +1,277 @@
+name: Launch Cursor Agent on PR Review
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)